Cargo.toml

```
@@ -38,7 +38,10 @@ exclude = [
 ]
 
 [workspace.package]
-version = "3.0.1"
+# version uses a `-nightly` on `main` branch so that snapshot releases cut from main will
+# display `-nightly` in the revision information of the running server. Releases cut from
+# a release branch will not use `-nightly`.
+version = "3.0.1-nightly"
```

```suggestion
version = "3.1.0-nightly"
```

This is going to turn into the 3.1 release so I think it should be the nightly for that.

---

That makes sense, so we want to `-nightly` whatever is the next minor release version. So, when `3.1.0` is released, we go to `3.2.0-nightly`. This is regardless of how many patch releases are done on the current minor version.

---

@pauldix I changed it to `3.1.0-nightly` and extended the comment for a bit more explanation in https://github.com/influxdata/influxdb/pull/26349/commits/3a9428c3708fae984f6b2a5510724000ff8b0249